lib/msal-browser/src/naa/BridgeProxy.ts

```diff
@@ -0,0 +1,230 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { AccountInfo } from "./AccountInfo";
+import {
+    AccountByHomeIdRequest,
+    AccountByLocalIdRequest,
+    AccountByUsernameRequest,
+} from "./AccountRequests";
+import { AuthBridge } from "./AuthBridge";
+import { BridgeCapabilities } from "./BridgeCapabilities";
+import { BridgeRequest } from "./BridgeRequest";
+import { BridgeRequestEnvelope, BridgeMethods } from "./BridgeRequestEnvelope";
+import { BridgeResponseEnvelope } from "./BridgeResponseEnvelope";
+import { IBridgeProxy } from "./IBridgeProxy";
+import { InitializeBridgeResponse } from "./InitializeBridgeResponse";
+import { TokenRequest } from "./TokenRequest";
+import { TokenResponse } from "./TokenResponse";
+
+declare global {
+    interface Window {
+        nestedAppAuthBridge: AuthBridge;
+    }
+}
+
+/**
+ * BridgeProxy
+ * Provides a proxy for accessing a bridge to a host app and/or
+ * platform broker
+ */
+export class BridgeProxy implements IBridgeProxy {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    static bridgeRequests: any[] = [];
+    static crypto: Crypto;
```

Comment:
Reason for static member rather than just accessing window.crypto directly when needed?